pkgs/populate/default.nix

```diff
@@ -161,9 +161,12 @@ let
         ${concatMapStringsSep " "
           (pattern: /* sh */ "--exclude ${quote pattern}")
           (config.exclude or [])} \
+        ${concatMapStringsSep " "
+          (filter: /* sh */ "--${filter.type} ${quote filter.pattern}")
+          (config.filters or [])} \
```

Comment:
This overlaps with the exclude option actually.

Reply:
I would either deprecate `exclude` or and add an include option while removing `filters`. Having both `filters` and `exclude` is confusing.